src/app/lib/config/services/amharic.js

```diff
@@ -129,36 +129,28 @@ const service = {
   footer: {
     externalLink: {
       href: 'https://www.bbc.co.uk/help/web/links/',
-      text: 'Read about our approach to external linking.',
+      text: 'ስለ ውጪ ሊንኮች ያለን አቀራረብ',
```

Comment:
```suggestion
      text: 'ስለ ውጪ ሊንኮች ያለን አቀራረብ.',
```

Reply:
@pjlee11 Amharic has different punctuation marks to English. `፡፡` should be used instead of `.`. Source: https://en.wikipedia.org/wiki/Amharic#Punctuation

Reply:
Good spot @sareh - The PAL links use fullstops 😞

Reply:
Interesting! I did wonder if punctuation marks were different to English. Do you think this is the same for Korean? @sareh

Reply:
Doing some searching online, I believe Korean uses the same `.` as Latin text.